nasa/openmct
# ESLint rules: rules requiring minor changes

Added the following rules to ESLint config and resolved issues found (10):

- "no-unused-expressions": "error"
- "no-useless-concat": "error"
- "radix": "error"
- "require-await": "error"
- "no-alert": "error"
- "no-useless-constructor": "error"
- "no-duplicate-imports": "error"

Resolves part of: https://github.com/nasa/openmct/issues/3059

**Testing instructions**

1. Make code changes violating above rules.
2. Run npm run lint.
3. See that violations of rules are observed.
4. Undo changes and see that violations are gone.
(note: violations should also be observed in editor)

**Author Checklist**

- Changes address original issue? Yes
- Unit tests included and/or updated with changes? N/A
- Command line build passes? Yes
- Changes have been smoke-tested? Yes
- Testing instructions included? Yes
